src/background/modules/ContextMenu.js

```diff
@@ -32,6 +63,12 @@ function handleMenuChoosen(info, tab) {
     text = text.normalize();
     const output = UnicodeTransformationHandler.transformText(text, info.menuItemId);
 
+    // Thunderbird workaround for https://bugzilla.mozilla.org/show_bug.cgi?id=1641575
+    if (info.fieldId) {
+        fallback(output, info.fieldId);
+        return;
```

Comment:
Note that's obviously not technically a real fallback. For it to be a fallback, we would have to try the original `executeScript`, catch an exception (or what happens when that fails?) and then try this fallback.

This would ensure that if it is ever fixed, it instantly works again.

If that is not possible I'm fine with how it is, I'm just saying/wanted to point that out.

Reply:
Good point. The insert does not actually fail, it just inserts the transformed text into the message body instead of the subject as the user was expecting. See your video for an example: https://github.com/rugk/unicodify/issues/73#issuecomment-1154584262. The bug was marked as WONTFIX, so I do not believe it will be fixed any time soon... Anyway, I am happy to rename this function if you have a better name.

Reply:
No all fine, just wanted to point that out.